systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeKafkaExternalListenersST.java

```diff
@@ -164,5 +166,8 @@ private void testWeirdUsername(String weirdUserName, KafkaListenerAuthentication
     @BeforeAll
     void createClassResources() throws Exception {
         deployClusterOperator(NAMESPACE);
+
+        kafkaBridgeClientJob = new KafkaBridgeClientsResource(producerName, consumerName, KafkaBridgeResources.serviceName(CLUSTER_NAME),
+            TOPIC_NAME, MESSAGE_COUNT, "", ClientUtils.generateRandomConsumerGroup(), bridgePort, 1000, 1000);
```

Comment:
This seems to be the same for all the Bridge tests. Can this be moved in some way to the abstract class?